js.js

```diff
@@ -0,0 +1,18 @@
+const body = document.querySelector('body');
```

Comment:
Should we wrap all of the code into iffy? 

https://codeburst.io/javascript-what-the-heck-is-an-immediately-invoked-function-expression-a0ed32b66c18

Reply:
done